### DISCUSSION TITLE
Notes don't follow the cursor with Aim Assist when No Scope (always hidden cursor) is enabled

### DISCUSSION THREAD

**Discussion by Snake-GGJJWP** [ppy/osu#16791] [Ideas] [answered]
#### **Problem**
When playing a map with `No Scope` (`always hidden` customisation) and `Aim Assist` mode objects don't follow the cursor. Instead they gravitate to the place where the cursor was last time it was visible. As far as I get it if `No Scope` with `always hidden` customisation is enabled the game remembers the cursor's last position when it was still visible. It can clearly be seen when paused: orange cursor where you need to click to continue doesn't match the cursor's real position before pausing. Also, this bug appears only if `No Scope` is enabled with `always hidden` cutomisation.
#### **Video**
https://youtu.be/E-NlLufS9Ck
#### **Version and logs**
osu!lazer 2022.205.0
[database.log](https://github.com/ppy/osu/files/8008677/database.log)
[performance.log](https://github.com/ppy/osu/files/8008678/performance.log)
[runtime.log](https://github.com/ppy/osu/files/8008679/runtime.log)
[updater.log](https://github.com/ppy/osu/files/8008681/updater.log)

**Comment by frenzibyte** [comment, marked as answer]
Tracking in https://github.com/ppy/osu/issues/16792.
